projects/schematics/src/migrations/mechanism/update-ssr/update-ssr-files_spec.ts

```diff
@@ -77,4 +78,34 @@ describe('Update SSR', () => {
       tree
     );
   });
+
+  describe('updateServerFile', () => {
+    it('should change nguniversal import in server.ts', async () => {
+      tree = await updateSsrSchematicRunner.runSchematic(
+        'update-ssr',
+        { name: 'schematics-test' },
+        tree
+      );
+
+      const updatedContent = tree.read('./server.ts')!.toString();
+      expect(updatedContent).toContain('@spartacus/setup/ssr');
+    });
+
+    it('should change zone.js import in server.ts', async () => {
+      let serverContent = tree.read('./server.ts')!.toString();
+
+      if (!serverContent.includes('zone.js')) {
+        serverContent = serverContent + `import "${OLD_ZONE_IMPORT}"`;
+        tree.overwrite('./server.ts', serverContent);
+      }
+      tree = await updateSsrSchematicRunner.runSchematic(
+        'update-ssr',
+        { name: 'schematics-test' },
+        tree
+      );
+
+      const updatedServerContent = tree.read('./server.ts')!.toString();
+      expect(updatedServerContent).toContain(NEW_ZONE_IMPORT);
```

Comment:
Maybe it's also worth checking whether the old import path has been removed.

Reply:
I've added an extra check for the old import path